File: SIMINOV-HYBRID-PHONEGAP-TEMPLATE/src/siminov/hybrid/phonegap/template/events/NotificationEventHandler.java
package siminov.hybrid.phonegap.template.events;

import siminov.connect.events.INotificationEvents;
import siminov.connect.exception.NotificationException;
import siminov.connect.notification.design.IMessage;
import siminov.connect.notification.design.IRegistration;

public class NotificationEventHandler implements INotificationEvents {

	public void onRegistration(IRegistration registration) {
		System.out.print("");
	}

	public void onUnregistration(IRegistration registration) {
		System.out.print("");
	}

	public void onNotification(IMessage message) {
		System.out.print("");
	}

	public void onError(NotificationException notificationException) {
		System.out.print("");
	}
}

File: SIMINOV-HYBRID-PHONEGAP-TEMPLATE/assets/www/Liquor/Events/NotificationEventHandler.js


function NotificationEventHandler() {
	
	this.onRegistration = function(registration) {
		//alert("onRegistration");
	}

	this.onUnregistration = function(registration) {
		//alert("onUnregistration");
	}

	this.onNotification = function(message) {
		//alert("onNotification");
	}
	
	this.onError = function(notificationException) {
		//alert("onError");
	}
}